service/rolesService.ts

```diff
@@ -0,0 +1,13 @@
+const axios = require('axios');
+import { Roles } from "../model/Roles";
+const API_BASE_URL: string='http://localhost:8080/api'
```

Comment:
URL's should be passed into the application via environment variables, as currently this won't work when deployed to AWS